packages/server/lib/socket.coffee

```diff
@@ -67,28 +70,23 @@ class Socket
     ## integration/foo_spec.js becomes cypress/my-integration-folder/foo_spec.js
     log("watch test file #{originalFilePath}")
     filePath = path.join(config.integrationFolder, originalFilePath.replace("integration/", ""))
-    filePath = filePath.replace("#{config.projectRoot}/", "")
+    filePath = path.relative(config.projectRoot, filePath)
```

Comment:
I wonder if this is going to fix watching specs on Windows OS

Reply:
Could be. That would be a freebie win.

Reply:
There was also a bug in browserify-preprocessor itself that affected watching regardless of OS (which I'm surprised wasn't reported outside of Windows). That's fixed as well with this PR (bump browserify-preprocessor to 1.0.1), so fingers crossed it takes care of the Windows issue.